internal/antewrapper/msg_fees_decorator_test.go

```diff
@@ -33,6 +33,28 @@ func (suite *AnteTestSuite) TestEnsureMempoolAndMsgFees() {
 	suite.Require().Contains(err.Error(), "Base Fee+additional fee cannot be paid with fee value passed in : \"100000atom\", required: \"100100atom\" = \"100000atom\"(base-fee) +\"100atom\"(additional-fees): insufficient fee", "got wrong message")
 }
 
+// checkTx true, high min gas price(high enough so that additional fee in same denom tips it over,
+//and this is what sets it apart from MempoolDecorator which has already been run)
+func (suite *AnteTestSuite) TestEnsureMempoolAndMsgFeesNoAdditionalFees() {
+	err, antehandler := setUpApp(suite, true, "nhash", 0)
+	tx, _ := createTestTx(suite, err, sdk.NewCoins(sdk.NewInt64Coin("nhash", 100000)))
+	suite.Require().NoError(err)
+
+	// Set gas price (1 atom)
```

Comment:
this says `atom` but code says `nhash`

Reply:
:doh: yes, i cleaned up the comments and added a new test.